terraform/modules/dbx_uc_create_catalog/main.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  name_postfix = var.env == "prod" ? "" : "-${var.env}"
+}
+
+resource "databricks_storage_credential" "external" {
+  name = "catalog-creds-gcs-${var.gcs_bucket_name}${local.name_postfix}"
+  databricks_gcp_service_account {}
+}
+
+resource "google_storage_bucket_iam_member" "member" {
+  bucket = var.gcs_bucket_name
+  role   = "roles/storage.admin"
+  member = "serviceAccount:${databricks_storage_credential.external.databricks_gcp_service_account}"
+}
+
+resource "databricks_catalog" "this" {
```

Comment:
Samme som for schema, burde vi opprette [grants](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/grants)?

Reply:
Burde få på plass at teamet er eier av katalogen.